src/components/BarChart/BarChart-container.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { BAR_CHART_WIDTH } from '../../theme/constants';
+import BarChartView from './BarChart-view';
+
+import classes from './BarChart.module.css';
+
+const BarChartContainer = () => {
+  const wallets: any = []; //TODO: import data 
+
+  const [activeSegment, updateActiveSegment] = useState(0);
+
+  const onClick = (index: number) => updateActiveSegment(index);
+
+  const getStyle: any = (acc: any, object: any) => ({
+    position: 'absolute',
+    height: '100%',
+    top: 0,
+    left: acc.position,
+    width: BAR_CHART_WIDTH * object.percentage / 100,
+  })
+
+  const getClasses = (index: any): any => {
+    const adjustedClasses = [classes.BarChartSegment]
+    const colors = [
+      'BarChartSegmentColorMarket', 
+      'BarChartSegmentColorPrivate', 
+      'BarChartSegmentColorDaap', 
+      'BarChartSegmentColorFraud'
+    ];
+
+    if (index < 10 && index === activeSegment) {
+      adjustedClasses.push(classes.BarChartSegmentActive);
+    }
+
+    if (!(index === activeSegment) && index === 0) {
+      adjustedClasses.push(classes.BarChartSegmentFirstInactive)
+    }
+
+    adjustedClasses.push(classes[colors[Math.round(Math.random() * 3)]])
+
+    return adjustedClasses.join(' ');
+  };
+
+  const data = wallets.reduce((acc: any, object: any, index: any) => {
+    if (!(object.percentage < 0.1)) {
```

Comment:
`object.percentage < 0.1` ?

Reply:
to reduce the number of others
![others](https://user-images.githubusercontent.com/14201464/65869858-6d8dc180-e37b-11e9-9681-a5c56372f9e0.jpg)

Reply:
I mean that it can be written as `object.percentage >= 0.1` instead of with additional negation